src/components/Profile.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import styled from "styled-components";
+
+export default function Profile({ profile }) {
+    const { id, name, age, role, univ, major, phoneNum, email, githubLink, imageUrl } = profile;
+
+    return (
+        <ProfileCard>
+            <Info>
+                <Detail>
+                    <div style={{
+                        fontSize: '2rem',
+                        fontWeight: 'bold',
+                        marginBottom: '1rem'
+                    }}>{name}({age})</div>
```

Comment:
styled-component 이용해주시면 코드가 더 깔끔해지고, 해당 태그가 어떤 역할을 하는지 파악하기 쉬워집니다!
변수 이름을 태그의 역할이 잘 녹아나게 지어주시면 좋습니다 :)
```suggestion
                    <Name>{name}({age})</Name>
                    
                    const Name = styled.p`
                      font-size: 2rem,
                      font-weight: bold,
                      margin-bottom: 1rem
                    `;
```